rubocop.yml

```diff
@@ -290,3 +302,15 @@ Style/OptionalBooleanParameter:
 
 Style/StringConcatenation:
   Enabled: false
+
+Style/CombinableLoops:
+  Enabled: true
+
+Style/KeywordParametersOrder:
+  Enabled: true
+
+Style/RedundantSelfAssignment:
+  Enabled: true
+
+Style/SoleNestedConditional:
+  Enabled: false
```

Comment:
Disabled this one... It's useful in theory, but in practice it tries to make code like this a one liner even though it's much easier to understand this way:

```ruby
if admin?
  return default_error unless user_in_staff_permissions_group?(value)
end
```